<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1040</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="w_categorie">
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="w_objetMedia">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>300</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nouvelle colonne</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nouvelle colonne</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nouvelle colonne</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="w_compteur"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionAjouter"/>
    <addaction name="actionSupprimer"/>
    <addaction name="actionModifier"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAction"/>
   <addaction name="menuOption"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSauvegarder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Sauvegarder</string>
   </property>
   <property name="toolTip">
    <string>Sauvegarder profil</string>
   </property>
   <property name="statusTip">
    <string>Sauvegarder profil</string>
   </property>
   <property name="whatsThis">
    <string>Sauvegarder profil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
   <property name="iconText">
    <string>Nouveau profil</string>
   </property>
   <property name="toolTip">
    <string>Nouveau profil</string>
   </property>
   <property name="statusTip">
    <string>Nouveau profil</string>
   </property>
   <property name="whatsThis">
    <string>Nouveau profil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>&amp;Charger</string>
   </property>
   <property name="toolTip">
    <string>Charger profil</string>
   </property>
   <property name="statusTip">
    <string>Charger profil</string>
   </property>
   <property name="whatsThis">
    <string>Charger profil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionAjouter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="iconText">
    <string>Ajouter une entrée</string>
   </property>
   <property name="toolTip">
    <string>Ajouter une entrée</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="iconText">
    <string>Supprimer une entrée</string>
   </property>
   <property name="toolTip">
    <string>Supprimer une entrée</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
